packages/j/jos-animation.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "jos-animation",
+  "description": "An animation library package to instantly add beautiful/professional looking animation to your website",
+  "homepage": "jos-animation.vercel.app",
+  "filename": "dist/jos.js",
```

```suggestion
  "filename": "jos.js",
```
As the `basePath` is set to `dist`, we'll pull the files out of that directory into the root.